conda/dev.yml

```
@@ -0,0 +1,43 @@
+# Conda development environment for testing local source code changes to zppy-interfaces before merging them to production (main branch).
+name: zppy-interfaces-dev
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  # Build
+  # =======================
+  - python >=3.9
+  - pip
+  - setuptools >= 60
+  # Base
+  # =================
+  - matplotlib-base >=3.8.2,<3.10
+  - mpas_tools >=0.21.0
+  - netcdf4
+  - numpy >=2.0,<3.0
+  - xarray >=2023.02.0
+  - xcdat >=0.7.0,<1.0
```

```suggestion
  - xcdat >=0.7.3,<1.0
```

xcdat v0.7.3 is now available on `conda-forge` with the same numpy 2.0 constraint: https://anaconda.org/conda-forge/xcdat